guides/v2.2/cloud/project/project-start.md

```diff
@@ -73,16 +73,16 @@ After pushing changes to your environment, you can observe the build results in
 You can review the logs from the deploy hook in the following directories:
 
 -  **Integration**: `/var/log/deploy.log`
--  **Staging**: `/var/log/platform/<project-ID>_stg/post_deploy.log`
--  **Production**: `/var/log/platform/<node-{1|2|3}>.<project-ID>/post_deploy.log`
+-  **Staging**: `/var/log/platform/<project-ID>_stg/deploy.log`
```

Comment:
@meker12  Please take a look at this wiki and confirm these changes: https://wiki.corp.magento.com/display/~yyevtushenko/Magento+Cloud+logs